livingdoc-converters/src/main/kotlin/org/livingdoc/converters/BooleanConverter.kt

```diff
@@ -8,7 +8,7 @@ import java.lang.reflect.AnnotatedElement
 open class BooleanConverter : TypeConverter<Boolean> {
 
     @Throws(ConversionException::class)
-    override fun convert(value: String, element: AnnotatedElement?): Boolean {
+    override fun convert(value: String, element: AnnotatedElement?, documentClass: Class<*>?): Boolean? {
```

Comment:
Why are all the type converters now returning nullable instances? I think a type converter should never return null. At least none of our default should...

Reply:
refactoring error
need to doublecheck changes made by the ide next time